lib/logger.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Utility functions for command line output logging from Protractor.
+ * May be used in different processes, since the launcher spawns
+ * child processes for each runner.
+ *
+ * All logging directly from Protractor, its driver providers, or runners,
+ * should go through this file so that it can be customized.
+ */
+export module Logger {
```

Comment:
When changing to `export namespace Logger {`, it appears that when doing a `var log = require('./logger')` would change the calls like `log.set` to  `log.Logger.set(` for javascript files that use it. Please advise.